frb_rust/Cargo.toml

```diff
@@ -16,9 +16,14 @@ threadpool = "1.8.1"
 lazy_static = "1.4.0"
 parking_lot = "0.12.1"
 flutter_rust_bridge_macros = { path = "../frb_macros", version = "1" }
+uuid = { version = "1.1.2", optional = true }
 
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
-allo-isolate = { version = "0.1.14-beta.2", features = ["anyhow", "chrono"] }
+allo-isolate = { version = "0.1.14-beta.3", features = [
```

Comment:
nit: replace with top-level [dependency features](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features) to allow users to opt out

Reply:
not sure, like 52c14600fddce107b36658b21bea024eebdd0d71 ?